docs/bnb_greenfield/api/blockchain-cli/bank.md

```diff
@@ -0,0 +1,173 @@
+---
```

Comment:
I think no need to list every sub command of chain and SP, redirect to greenfield repo and sp command repo?

Reply:
Done. Removed them to greenfield chain & sp repos